src/Components/list.js

```diff
@@ -14,13 +15,25 @@ function List() {
         setLoading(false)
     }, [month])
 
+    const handleMonthChange = (e) => {
+        setMonth(e.target.value)
+    }
+
     if (loading) {
         return 'Loading'
     }
 
+    const noFruitsInThisMonths = fruits.length === 0 
```

Comment:
Maybe change `months` for `month`?

Reply:
Ok, I'm going to change it now .. thanks!